### PR TITLE
Fix/1815 change owner for all actions on content

### PR DESCRIPTION
**Pull request by inkhey**
related to #1815

## Checkpoints
- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME -> No fixme
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present -> No fixme
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation -> associated test to this fixture now check also owner change for case copy/move/set_status

### CODE REVIEW

**Comment by inkhey** [comment]
@Skylsmoi integration test failed here for unknown reason. That may mean that frontend expect uncorrect behaviour this issue fix.

File: backend/tracim_backend/lib/core/content.py [hunk 1]
```diff
@@ -1405,7 +1406,7 @@ def _move_current(
         if new_parent and new_parent != item.parent:
             content_type = content_type_list.get_one_by_slug(item.type)
             self._check_valid_content_type_in_dir(content_type, new_parent, new_workspace)
-
+        item.owner = self._user
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: I think `self._user` may be `None`. Please check, and if so, then the code should be `item.owner = self._user if self._user`

File: backend/tracim_backend/lib/core/content.py [hunk 2]
```diff
@@ -1641,6 +1642,7 @@ def _add_copy_revisions(
             content=new_content,
             force_create_new_revision=True,
         ) as rev:
+            rev.owner = self._user
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: `rev.owner = self._user if self._user`

File: backend/tracim_backend/lib/core/content.py [hunk 3]
```diff
@@ -1362,6 +1362,7 @@ def restore_content_default_allowed_content(self, content: Content) -> None:
 
     def set_status(self, content: Content, new_status: str):
         if new_status in content_status_list.get_all_slugs_values():
+            content.owner = self._user
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: `content.owner = self._user if self._user`

**Comment by inkhey** [comment]
> I think you did not fix the bug in case the content or label are modified

Issue didn't happened in thoses case.

File: backend/tracim_backend/lib/core/content.py (hunk 3 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: done

File: backend/tracim_backend/lib/core/content.py (hunk 2 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: done

File: backend/tracim_backend/lib/core/content.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: this syntax is not correct , replace by : 
```
if self._user:
  item.owner = self._user
```

Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: done

**Comment by buxx** [comment]
Travis pipelines are green (but github say "in progress", i think it is sticked). So i merge.

**Pull request merged**